packages/components/pie-modal/src/index.ts

```diff
@@ -115,6 +118,24 @@ export class PieModal extends RtlMixin(LitElement) {
         }
     }
 
+    /**
+     * Return focus to the specified element, providing the selector is valid
+     * and the chosen element can be found.
+     * Fails silently.
+     */
+    private _returnFocus () {
+        const selector = this.returnFocusAfterCloseSelector?.trim();
+
+        if (selector) {
+            try {
+                const focalPoint = document.querySelector(selector) as HTMLElement;
+                focalPoint?.focus();
+            } catch (ignore) {
+                // Not necessary to throw an error here.
+            }
```

Comment:
In more elaborate cases I have seen this rationale reduce the footprint of the code significantly.

Reply:
i believe the try/catch here isn't needed as none of the logic will throw

Reply:
So that method will throw in a couple of scenarios:
1. An empty string is passed in (although I catch this with `.trim()`
2. An invalid selector is passed in, e.g., `h1[`.

In case 2 I think we probably do want to throw an error so that the consuming app can fix whatever they're doing. What do you think?

In terms of the type assertion I'm doing here, `as HTMLElement`, that is actually necessary because `focus` doesn't exist on `Element` but it does on `HTMLElement`. Is that what you were referring to or was that more to do with the handling of the error?